web/src/components/speaker/SpeakerPagination.vue

```diff
@@ -0,0 +1,55 @@
+<template>
```

Comment:
Perhaps rename this file to PaginationRow or PaginationLayout, to keep it abstract in case we use it more places later?

Reply:
@ann-kilzer updated here:
https://github.com/WWCodeTokyo/speak-her-db/pull/186/commits/3ea36be6942c9df8c270a9db3fed6887f64681aa